### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by connormck95** [mtoto/ufc.stats#9]
Hey love these stats! Will an update be ran again?

**Comment by resinger-am** [comment]
Agreed, love this database!!!  Will this be updated?  I am a newbie, but have connected to the data and will do whatever I can to help keep this updated, if you need any help doing so.
